src/Judge/NTS.Judge.MAUI/installer-script.iss

```diff
@@ -0,0 +1,132 @@
+#define AppExe ".\\bin\\Release\\net8.0-windows10.0.19041.0\\win-x64\\publish\\NTS.Judge.MAUI.exe"
```

Comment:
See if these paths can use wildcards in attempt to remove the specific windows build from the script